src/authorize.ts

```diff
@@ -0,0 +1,152 @@
+import * as express from 'express';
+
+import * as session from './session';
+
+import debugModule from 'debug';
+const debug = debugModule('everyauth:authorize');
+
+export interface IEveryAuthContext {
+  /** The url which the service was configured to use. */
+  finishedUrl: string;
```

Comment:
I think this should be called callbackUrl instead.

Reply:
Keeping as finishedUrl for the moment.